drivers/hmis/app/graphql/mutations/create_assessment.rb

```diff
@@ -0,0 +1,63 @@
+module Mutations
+  class CreateAssessment < BaseMutation
+    argument :enrollment_id, ID, required: true
+    argument :assessment_role, String, required: true
```

Comment:
In the doc we had this as an enum with values `INTAKE|UPDATE|ANNUAL|EXIT|CE|POST_EXIT|CUSTOM`, and then later on we could add a string field for `customRoleIdentifier` or something like that, when we get to supporting custom assessments. We could also can add others as needed, like `MOVE_IN` if we want to do move-in assessments like this.

I prefer using an enum instead of a string value, so that the list of possible `assessment_role`s lives in the backend code instead of the frontend code. @eanders @ZachCMP ?

Reply:
yeah

Reply:
Fixed